model.py

```diff
@@ -38,14 +41,30 @@ def __init__(self, username, password):
     def __repr__(self):
         """Provide helpful representation when printed."""
 
-        return f"<User user_id={self.user_id} username={self.username}>"
+        return f"<User {self.id} | {self.username}>"
 
     @classmethod
     def get_user_by_username(cls, username):
         """Get a user, given their username."""
 
         return cls.query.filter_by(username=username).one()
 
+    def save(self):
+        """A method to make saving users simpler."""
+        db.session.add(self)
+        db.session.commit()
+
+    def create_password(self, password):
```

Comment:
Easy to secure our passwords!

Also see the addition of a  `save()` method. It's a good example of how you can streamline your models.